docs/tutorials/getting-started-rails.txt

```diff
@@ -0,0 +1,515 @@
+=======================
+Getting Started (Rails)
+=======================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+New Application
+---------------
+
+This section shows how to create a new Ruby on Rails application using Mongoid
+for data access. The application will be similar to the blog application
+described in the `Ruby on Rails Getting Started
+<https://guides.rubyonrails.org/getting_started.html#creating-the-blog-application>`_
+guide, however using Mongoid instead of ActiveRecord as the database adapter.
+
+
+Install ``rails``
+*****************
+
+We will use a Rails generator to create the application skeleton.
+In order to do so, he first step is to install the ``rails`` gem:
+
+.. code-block:: sh
+
+    gem install rails -v '~> 6.0.0'
+
+
+Create New Application
+**********************
+
+Use the ``rails`` command to create the application skeleton, as follows:
+
+.. code-block:: sh
+
+    rails new blog --skip-active-record --skip-bundle
+    cd blog
+
+.. note::
+
+    You may receive a warning like this:
+
+    .. code-block:: sh
+
+        Could not find gem 'puma (~> 3.11)' in any of the gem sources listed in your Gemfile.
+        Run `bundle install` to install missing gems.
+        
+    Disregard it as we will be taking care of gem installation
+    in a moment.
+
+We pass ``--skip-active-record`` to request that ActiveRecord is not added
```

Comment:
You might want to briefly explain what ActiveRecord is, for the people who aren't familiar with Rails.

Reply:
I must say I never thought this to be an issue, but what language would you suggest?

Reply:
If you think this isn't a common situation, then feel free to disregard.

I was thinking something like:
> ActiveRecord is an object relational mapper (ORM) that is included by default in Ruby on Rails projects. Because we are using Mongoid in this project, ActiveRecord is not required, and we pass `--skip-active-record` to request that it not be added.